itou/job_applications/models.py

```diff
@@ -838,11 +838,23 @@ def refused_by(self):
     def can_be_cancelled(self):
         if self.origin == Origin.AI_STOCK:
             return False
-        if not self.employee_record.exists():
-            # A job application can be canceled provided that there is no employee record linked to it,
-            # as it is possible that some information were already sent to the ASP.
-            return True
-        return False
+
+        # If a job application is linked to an employee record then it can only be cancelled if we didn't send any data
+        employee_record_is_blocking_cancellation = self.employee_record and any(
```

Comment:
```suggestion
        employee_record_is_blocking_cancellation = any(
```
devrait marcher non ?